api/db/database-builder/factory/build-sco-blocked-access-dates.js

```diff
@@ -0,0 +1,28 @@
+import { databaseBuffer } from '../database-buffer.js';
+
+const buildScoBlockedAccessDates = function ({
+  collegeDate = new Date('2025-11-15'),
+  lyceeDate = new Date('2025-10-15'),
+} = {}) {
+  databaseBuffer.pushInsertable({
+    tableName: 'sco_blocked_access_dates',
+    values: {
+      scoOrganizationType: 'college',
```

Comment:
Je propose d'aligner sur des valeurs representatives, et qui sont celle utilisees dans Pix (enum dans `api/src/organizational-entities/domain/models/Tag.js`)


```suggestion
      scoOrganizationType: 'COLLEGE',
```

Reply:
C'est fait :+1: